pkg/controller/master/upgrade/common.go

```diff
@@ -781,6 +781,11 @@ func upgradeReference(upgrade *harvesterv1.Upgrade) metav1.OwnerReference {
 	}
 }
 
+func removeItemFromSlice(slice []string, i int) []string {
```

Comment:
Can you help add a doc comment to the function and explain its feature and result order change

Reply:
for long term we can add utility functions in https://github.com/harvester/go-common/blob/main/slice.go too

Reply:
Updated, thank you!